src/json-editor.component.ts

```diff
@@ -73,6 +73,7 @@ export class JsonEditorComponent extends AbstractTrackerComponent implements OnI
   previews: Array<Preview> = [];
   isPreviewerHidden: boolean;
   keys: Set<string>;
+  paths: PathCache = {};
```

Comment:
`pathCache` might be better I personally use plurals when the variable is an array.